packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/sharedobjects/SharedObject.kt

```diff
@@ -19,6 +19,11 @@ open class SharedObject(runtimeContext: RuntimeContext? = null) {
    */
   internal var sharedObjectId: SharedObjectId = SharedObjectId(0)
 
+  // Used by JNI
+  private fun getSharedObjectId(): Int {
```

Comment:
```suggestion
  @Suppress("unused")
  @DoNotStrip
  private fun getSharedObjectId(): Int {
```

maybe DoNotStrip is not necessary? since the class has DoNotStrip already